R/mod_review_forms.R

```diff
@@ -231,8 +227,8 @@ mod_review_forms_server <- function(
       review_save_error(FALSE)
       golem::cat_dev("Save review status reviewed:", input$form_reviewed, "\n")
       
-      review_row <- review_data_active() |> 
-        dplyr::distinct(subject_id, item_group) |> 
+      review_records <- review_data_active() |> 
+        dplyr::distinct(id) |>
```

Comment:
Just wondering why to use `distinct()`. I thought `id` can never be duplicated with how it is created?

Reply:
I think this just got left in when I was updating code and didn't get around to removing it.